rpc/internal/perf_service.go

```diff
@@ -61,6 +63,12 @@ func (srv *perfService) CreateMetricSeries(ctx context.Context, result *ResultDa
 		return resp, errors.Wrap(err, "problem creating ftdc rollups job")
 	}
 
+	if len(record.Rollups.Stats) > 0 {
+		if err := srv.updateDownstreamPerfServices(ctx, record); err != nil {
+			return nil, errors.Wrap(err, "problem in updating downstream services")
```

Comment:
For error messages you can omit "problem" and just write here, for example, `updating downstream services`. We previously would begin them with "problem" but it is redundant since it is already and error message.